client/rpc/block.go

```diff
@@ -17,12 +18,32 @@ import (
 )
 
 //BlockCommand returns the verified block data for a given heights
-func BlockCommand() *cobra.Command {
+func BlockCommand(cdc *codec.Codec) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "block [height]",
 		Short: "Get verified data for a the block at given height",
 		Args:  cobra.MaximumNArgs(1),
-		RunE:  printBlock,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			var height *int64
+			// optional height
+			if len(args) > 0 {
+				h, err := strconv.Atoi(args[0])
+				if err != nil {
+					return err
+				}
+				if h > 0 {
+					tmp := int64(h)
+					height = &tmp
+				}
+			}
+
+			output, err := getBlock(context.NewCLIContext(cdc), height)
+			if err != nil {
+				return err
+			}
+			fmt.Println(string(output))
```

Comment:
Need to implement some `fmt.Stringer` on the tendermint side for these commands.